defaults/main.yml

```diff
@@ -31,9 +31,10 @@ nimbus_fluffy_user: 'nimbus'
 nimbus_fluffy_group: 'staff'
 
 # General
-nimbus_fluffy_network: 'mainnet' # or testnet0
+nimbus_fluffy_network: 'mainnet'
 nimbus_fluffy_log_level: 'INFO' # TRACE DEBUG INFO NOTICE WARN ERROR FATAL NONE
 nimbus_fluffy_bootstrap_nodes: []
+nimbus_fluffy_portal_network: 'testnet0' # or 'none'
```

Comment:
In Discord I meant that we should use a separate variable for the service name, not in the role.